### ISSUE TITLE
rename to stac-process?

### ISSUE THREAD

**Issue by matthewhanson** [stac-utils/stac-task#21]
The name stac-task may be misleading due to the use of "tasking" with regard to ordering future collects of data from a satellite or remote platform. See https://github.com/Element84/stat-api-spec

In this context a "task" is a processing function that takes in 0 or more STAC Items and returns 1 or more STAC Items.  Where used the term workflows refers to a series of tasks chained together.

Suggest renaming repo to stac-process to better reflect what it does.

**Comment by ircwaves** [comment]
After discussion at the last `stac-utils working group` meeting, it was decided that `stac-workflow` is what this should be re-named to.  To minimize disruptions, the following plan of action is proposed:
1. fork this repo
2. update the README here to inform folks that dev here has ceased, and they should use `stac-workflow`
3. Have more conversations about adding a `Workflow` class.

Does that jive with your recollections @matthewhanson and/or @gadomski ?

**Comment by gadomski** [comment]
Yeah, sounds good to me. IIRC there was also the suggestion to cut one more release after updating the README, so the PyPI page has a pointer to the new repo as well. I think the list should be:

- [ ] Fork this repo to **stac-workflow**
- [ ] Update **stac-workflow** to work (update imports, documentation, etc), and add a "History" section describing the migration from **stac-task**
- [ ] Release a version of **stac-workflow**
- [ ] Update the README here at **stac-task** to describe the new state of play
- [ ] Release a final version of **stac-task**
- [ ] Archive this repository
- [ ] ...
- [ ] Profit?!?

**Comment by jsignell** [comment]
Sorry to be revisiting an old decision but it seems easier to rename the repo. The benefit of that approach is mostly that github will preserve the redirect. It'll also preserve the stars and whatever other project-metadata is tracked by github outside of git.
